bats/tests/helpers/vm.bash

```diff
@@ -17,27 +17,29 @@ factory_reset() {
         sudo iptables -F
         sudo iptables -L | awk '/^Chain CNI/ {print $2}' | xargs -l sudo iptables -X
     fi
+}
+
+start_container_engine() {
+    # TODO why is --path option required for Windows
+    if is_windows; then
+        set - --path "$(wslpath -w "$PATH_EXECUTABLE")" "$@"
+    fi
+    if is_unix; then
+        set - --application.admin-access=false "$@"
+        set - --application.path-management-strategy rcfiles "$@"
+    fi
 
+    # TODO containerEngine.allowedImages.patterns and WSL.integrations
+    # TODO cannot be set from the commandline yet
     image_allow_list="$(bool "$RD_USE_IMAGE_ALLOW_LIST")"
-    path_management="rcfiles"
     wsl_integrations="{}"
     if is_windows; then
-        path_management="notset"
         wsl_integrations="{\"$WSL_DISTRO_NAME\":true}"
     fi
-
     mkdir -p "$PATH_CONFIG"
     cat <<EOF >"$PATH_CONFIG_FILE"
 {
   "version": 5,
-  "application": {
-    "adminAccess":            false,
-    "pathManagementStrategy": "$path_management",
-    "updater":                { "enabled": false }
-  },
-  "virtualMachine": {
-    "memoryInGB": 6
-  },
   "WSL": { "integrations": $wsl_integrations },
   "containerEngine": {
```

Comment:
why did you keep the allowed images empty? i believe this field can be removed.

Reply:
It is not empty (see below). The `patterns` are always set to `["docker.io"]`, and `enabled` is set according to the value of `RD_USE_IMAGE_ALLOW_LIST`. That env variable needs to be renamed, but not in this PR.